docs/Contracts/initializeContract.md

```diff
@@ -1,104 +0,0 @@
----
-sidebar_position: 1
----
-
-# Initialize Contract Object
-
-Before jumping into any functionality, a Contract object has to be initialized in the Toolbox.
-This is done with the `initContract` function.
-
-## Parameters
-
-The parameters of `initContract` function are described as follows.
-
-| Name         | Type   | Description                                                                                                                                           |
-| ------------ | ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `name`       | string | Name of the Contract used on deployment                                                                                                               |
-| `symbol`     | string | Symbol of the Contract used on deployment                                                                                                             |
-| `dir`        | string | Path to directory where the new Contract File is to be created                                                                                        |
-| `standard`   | string | `"ERC721"` or `"ERC1155"`                                                                                                                             |
-| `connection` | object | Used to set up **ethers.js** connection. Described [here](/docs/Contracts/initializeContract#connection-details)                                      |
-| `deployed`   | object | _(optional)_ Used for initialization of deployed contracts. Described [here](/docs/Contracts/initializeContract#initialization-for-deployed-contract) |
-
-### Connection Details
-
-The fields of `connection` parameter are described as follows.
-
-| Name       | Type   | Description                                                  |
-| ---------- | ------ | ------------------------------------------------------------ |
-| `network`  | string | The Blockchain network on which contract is (to be) deployed |
-| `provider` | object | Details of the JSON RPC provider to be used                  |
-| `wallet`   | object | Contains `privateKey` of wallet to be used on the network    |
-
-:::info
-The contents of `connection` parameter in `initContract` function are used to set up an
-[ethers.js](https://ethers.org/) connection.
-See their [documentation](https://docs.ethers.io/v5/) for more details.
-:::
-
-#### Supported Networks
-
--   Ethereum Mainnet (`"homestead"`)
```

Comment:
are these information documented elsewhere?

Reply:
yes, it is there in `documentation/docs/Contracts/initializeContract.md` as well